dags/inference/configs/jetstream_pytorch_inference.py

```diff
@@ -0,0 +1,117 @@
+"""A DAG to run MaxText inference benchmarks with nightly version."""
+
+import datetime
+from airflow import models
+from airflow.models.baseoperator import chain
+from dags import composer_env, test_owner
+from dags.vm_resource import TpuVersion, Zone, Project, V5_NETWORKS, V5E_SUBNETWORKS, V5P_SUBNETWORKS, RuntimeVersion
+from dags.inference.configs import jetstream_pytorch_gce_config
+from dags.multipod.configs.common import SetupMode, Platform
+
+
+# Run once a day at 4 am UTC (8 pm PST)
+SCHEDULED_TIME = "0 4 * * *" if composer_env.is_prod_env() else None
+
+
+with models.DAG(
+    dag_id="jetstream_pytorch_inference",
+    schedule=None,
+    tags=["inference_team", "jetstream_pytorch", "maxtext", "nightly"],
+    start_date=datetime.datetime(2024, 1, 19),
+    catchup=False,
+) as dag:
+  test_name_prefix = "jetstream-pytorch-inference"
+  test_models = {
+      "llama3-8b": {
+          "model_name": "llama-3",
+          "size": "8b", 
+          "sleep_time": 120,
+          "tpu_version_cores": [(TpuVersion.V5E, 8)],
+          "checkpoint": "gs://inference-benchmarks/models/llama3-8b",
+          "dataset": "openorca",
+          "tokenizer": "tokenizer.model",
+          "batch_sizes": [128],
+          "request_rate": 5,
+          "num_prompts": 200,
+          "max_output_length": 1024,
+          "max_cache_length": 2048, 
+          "sharding_config": "default_shardings/llama.yaml",
+      },
+      "llama2-7b": {
+          "model_name": "llama-2",
+          "size": "7b", 
+          "sleep_time": 120,
+          "tpu_version_cores": [(TpuVersion.V5E, 8)],
+          "checkpoint": "gs://inference-benchmarks/models/llama2-7b/pytorch/llama-2-7b/",
+          "dataset": "openorca",
+          "tokenizer": "tokenizer.llama2",
+          "batch_sizes": [128],
+          "request_rate": 5,
+          "num_prompts": 200,
+          "max_output_length": 1024,
+          "max_cache_length": 2048, 
+          "sharding_config": "default_shardings/llama.yaml",
+      },
+      "llama2-13b": {
+          "model_name": "llama-2",
+          "size": "13b", 
+          "sleep_time": 120,
+          "tpu_version_cores": [(TpuVersion.V5E, 8)],
+          "checkpoint": "gs://inference-benchmarks/models/llama2-13b/pytorch/llama-2-13b/",
+          "dataset": "openorca",
+          "tokenizer": "tokenizer.llama2",
+          "batch_sizes": [64],
+          "request_rate": 5,
```

Comment:
Nit: In general, the bigger model's request rate is smaller.

Reply:
Changed to 4